src/sedcli_main.c

```diff
@@ -199,6 +219,32 @@ static void echo_enable()
 	tcsetattr(1, 0, &term);
 }
 
+enum sed_print_flags {
+	SED_NORMAL = 1 << 0,
```

Comment:
just a curiosity question. why is bitfield? do you expect 2 format to be used at one time, through( SED_NORMAL|SED_UDEV)?

Reply:
Nope... This will be easy to add new print format types and use them in switch cases to print in corresponding formats

Reply:
I see if the intent is to be less complex, my recommendation is to use integers (enum) instead of bitfield